tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -331,19 +331,19 @@ def test_getUsers(self):
                                   [field % ('user_3', 'user_3', '', '43200')],
                                   '(UserName, UserDN, Pem, ExpirationTime)')]:
       result = db._update('INSERT INTO %s%s VALUES %s ;' % (table, fields, ', '.join(values)))
-      self.assertTrue(result['OK'], '\n%s' % result.get('Message') or 'Error message is absent.')
+      self.assertTrue(result['OK'], '\n' + result.get('Message', 'Error message is absent.'))
     for user, exp, expect, log in [(False, 0, ['user_1', 'user_2', 'user_3'], '\n* Without arguments'),
                                    (False, 1200, ['user_3'], '* Request proxy live time'),
                                    ('user_2', 0, ['user_2'], '* Request user name'),
                                    ('no_user', 0, [], '* Request not exist user name')]:
       gLogger.info('%s..' % log)
       result = db.getUsers(validSecondsLeft=exp, userMask=user)
-      self.assertTrue(result['OK'], '\n%s' % result.get('Message') or 'Error message is absent.')
+      self.assertTrue(result['OK'], '\n' + result.get('Message', 'Error message is absent.'))
       usersList = []
       for line in result['Value']:
         if line['Name'] in ['user_1', 'user_2', 'user_3']:
           usersList.append(line['Name'])
-      self.assertEqual(set(expect), set(usersList), '%s, when expected %s' % (usersList, expect))
+      self.assertEqual(set(expect), set(usersList), set(usersList) + ', when expected ' + set(expect))
```

Comment:
This doesn't work, unless you cast the `set`s to  `str` in the concatenation

Reply:
Oh, yes. I meant str(usersList) + .. + str(expect)...  I'll fix this

Reply:
done